R-packages/covidcast/R/plot.R

```diff
@@ -713,60 +697,60 @@ plot_line = function(x, range = NULL, title = NULL, params = list()) {
 }
 
 
-get_pr_centroid = function(map_df){
-  pr_df = map_df %>% dplyr::filter(.$is_pr) %>% sf::st_transform(., 102003)
-  pr_centroid = sf::st_centroid(sf::st_geometry(pr_df))
-  return(pr_centroid)
-}
 
+# Use the following CRS values
+
+final_crs = '+proj=aea +lat_1=29.5 +lat_2=45.5 +lat_0=37.5 +lon_0=-96 +x_0=0 
+             +y_0=0 +ellps=GRS80 +datum=NAD83 +units=m +no_defs'
+alaska_crs = '+proj=aea +lat_1=55 +lat_2=65 +lat_0=50 +lon_0=-154 +x_0=0 
+              +y_0=0 +ellps=GRS80 +datum=NAD83 +units=m +no_defs'
+hawaii_crs = '+proj=aea +lat_1=8 +lat_2=18 +lat_0=13 +lon_0=-157 +x_0=0 
+              +y_0=0 +ellps=GRS80 +datum=NAD83 +units=m +no_defs'
 
-shift_pr = function(map_df, pr_centroid){
+shift_pr = function(map_df){
   pr_df = map_df %>% dplyr::filter(.$is_pr)
-  pr_df = sf::st_transform(pr_df, 102003)
+  pr_df = sf::st_transform(pr_df, final_crs)
   pr_shift = sf::st_geometry(pr_df) + c(-0.9e+6, 1e+6)
   pr_df = sf::st_set_geometry(pr_df, pr_shift)
-  r = -16 * pi / 180
+  r = 16 * pi / 180
   rotation = matrix(c(cos(r), sin(r), -sin(r), cos(r)), nrow = 2, ncol = 2)
-  pr_rotate = (sf::st_geometry(pr_df) - pr_centroid) * rotation + pr_centroid
+  pr_rotate = (sf::st_geometry(pr_df) - c(0,0)) * rotation + c(0,0)
```

Comment:
is it actually necessary to add and subtract 0 here?

Reply:
Nope.  Removed!